apps/app.go

```diff
@@ -167,10 +171,12 @@ var DefaultBindingsCall = &Call{
 	Path: DefaultBindingsCallPath,
 }
 
+// App describes an App installed on a Mattermost instance. App should be
+// abbreviated as `app`.
 type App struct {
-	AppID    AppID     `json:"app_id"`
-	Manifest *Manifest `json:"manifest"`
-	Status   AppStatus `json:"app_status"`
+	Manifest
+
+	Disabled bool `json:"disabled,omitempty"`
```

Comment:
nit: use `Enabled` instead to be in line with `MarketplaceApp`, etc.?

Reply:
This was deliberate: meaningful nil defaults, by default (a newly created) App should be enabled, unless explicitly disabled. In UX "enabled" makes more sense.